### DISCUSSION TITLE
Windows : preventing the slint app to show in the Taskbar

### DISCUSSION THREAD

**Discussion by Sulquendi** [slint-ui/slint#4799] [Q&A] [answered]
Hi, 
any API allowing to toggle the slint app visibility in the Windows Taskbar ?

Regards

**Comment by bjorn** [comment, marked as answer]
Duplicate of #3266, where you will find some information.

You can try hooking into [`with_skip_taskbar`](https://docs.rs/winit/latest/i686-pc-windows-msvc/winit/platform/windows/trait.WindowBuilderExtWindows.html#tymethod.with_skip_taskbar) from the `window_builder_hook` used in https://github.com/slint-ui/slint/discussions/4284#discussioncomment-8077128.

**Reply by Horbin-Magician** [reply]
I would like to ask if there is currently a way to have one window hidden in the Taskbar (as a toolbar) and the other shown (as the main window)?
If use [with_skip_taskbar](https://docs.rs/winit/latest/i686-pc-windows-msvc/winit/platform/windows/trait.WindowBuilderExtWindows.html#tymethod.with_skip_taskbar), all windows of my app will be hiding in the Taskbar.

**Reply by bjorn** [reply]
I don't believe that's possible with the current API. Maybe the window builder hook could take the desired value from some other state in your application? Though maybe the window builder hook is only used once and not for every window? Maybe @ogoffart can help.

**Comment by Sulquendi** [comment]
Thank you, 100% Working
